src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -308,7 +308,8 @@ def add_to_nwbfile(
 
         # Be sure to copy metadata at this step to avoid mutating in-place
         videos_metadata = deepcopy(metadata).get("Behavior", dict()).get("ExternalVideos", None)
-        if videos_metadata is None:
+        # If no metadata is provided use the default metadata
+        if videos_metadata is None or self.video_name not in videos_metadata:
```

Comment:
When is this going to happen?

Reply:
Metadata with entries for `ExternalVideos` is received as an input but there is no entry for this specific video.

Reply:
Wait that sounds wrong.

Reply:
Forget it, that seems correct to me.